src/ngx_pagespeed.cc

```diff
@@ -607,6 +607,16 @@ ngx_int_t copy_response_headers_to_ngx(
       CHECK(pagespeed_headers.FindContentLength(&len));
       headers_out->content_length_n = len;
       headers_out->content_length = header;
+    } else if (STR_EQ_LITERAL(name, "Content-Encoding")) {
+      headers_out->content_encoding = header;
```

Comment:
I wonder if we should look at `r->gzip_vary` and any pre-existing `Vary: Accept-Encoding` headers here. Does PSOL also add a `Vary` header when it emits a `Content-Encoding` header?